### PR TITLE
Alert pages action: add new pages

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: .github/workflows/alert-docs.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ jobs:
         git checkout -b update-alerts
         # Update the index to be sure git is aware of changes
         git update-index -q --refresh
+        git add site/content/docs/alerts/*
```
Comment (kingthorin) [inline .github/workflows/alert-docs.yml:36]: Could just change L40 to add `site/*`?

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Oops that shoulda been comment not approve.

File: .github/workflows/alert-docs.yml (hunk 1 above)
Comment (psiinon) [reply .github/workflows/alert-docs.yml:36]: I prefer being more specific tbh - this script should only update pages in this subdirectory...

Comment (thc202) [reply .github/workflows/alert-docs.yml:36]: That would be too late, that one should no longer be needed.

Comment (kingthorin) [reply .github/workflows/alert-docs.yml:36]: Okay, include this at L40 then for consistency? Only worth adding if there's actually a change.

Comment (thc202) [reply .github/workflows/alert-docs.yml:36]: I'd remove the other one but +1 to keep them the same.

Comment (kingthorin) [reply .github/workflows/alert-docs.yml:36]: It shouldn't be too late, the line right before this updates the index so the if should be fine.

Comment (thc202) [reply .github/workflows/alert-docs.yml:36]: Saying it's late because that didn't work with the SSTI rules which were just new files (and why the change).

**Review by thc202 — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Pull request merged**